### ISSUE TITLE
Support deleting downloaded YouTube items

### ISSUE THREAD

**Issue by stuporglue** [ondras/cyp#24]
It would be nice if there were a way to delete downloaded YouTube items. 

My users can access cyp through the web, but don't have disk access to manage files. 

I think this could/should only apply to YouTube files. Since cyp is responsible for downloading them, I see no problem with it also having permissions to delete them.

**Comment by ondras** [comment]
Hmm, this might be pretty complex. We have no (UI) way of listing contents of the `_youtube` directory. One could argue that this directory shall be available via the MPD library, but there are no guarantees about its name and/or location within the MPD library structure.

**Comment by stuporglue** [comment]
No problem to mark this as a "won't fix" issue then. I thought it might be (relatively) simple since everything downloads to the _youtube directory, but since there's no existing code to do directory listings etc. it might be too much work to be worth it.

**Comment by rominator1983** [comment]
That is true of course. Though most things work when switching to yt-dlp by changing only the command line (this can be done with cyp too) since command line switches seem to be compatible. Interestingly enough the `-g` switch which makes it extract an audio stream URL  is not even documented in yt-dlp but works nonetheless.

**Comment by rominator1983** [comment]
Have played around a little. The switch -g is there but it's not work for longer media any more. At least for YouTube. Other sites might work still. I think at the time of the first ad it breaks and playback stops. Might still work with a premium account.
